docs/advanced-features/debugging.md

```diff
@@ -72,7 +72,7 @@ If you're using `npm run dev` or `yarn dev` (see [Getting Started](/docs/getting
 
 Launching the Next.js dev server with the `--inspect` flag will look something like this:
 
-```shellsession
+```bash
```

Comment:
```suggestion
```sh
```
Should we just us sh here?

Reply:
I compared usage of `sh` and `bash`

- sh: 25 (0 usage in docs)
- bash: 447